motr/m0crate/crate_index.c

```diff
@@ -343,6 +345,70 @@ static bool cr_idx_w_rebalance_ops(struct cr_idx_w *w, enum cr_opcode op);
 static int cr_idx_w_common(struct cr_idx_w *w);
 static int cr_idx_w_warmup(struct cr_idx_w *w);
 
+/** RESULT REPORT */
+static double cr_time_in_seconds(m0_time_t mtime){
+	return (m0_time_seconds(mtime)) +
+ 		( m0_time_nanoseconds(mtime) / (double) M0_TIME_ONE_SECOND );
+
+}
+static void cr_time_measure_report(struct cr_time_measure_ctx *t,
+				   	struct cr_idx_w w)
+{
+	int op_count = 0;
+	fprintf(stdout, "result: total_s, %f, avg_time_per_op_ns, %"PRIu64","
+		" key_size_bytes, %d, value_size_bytes, %d, ops, %d\n", 
+		t->elapsed, m0_time_nanoseconds(t->test_time) / w.nr_ops_total,
+		w.wit->key_size, w.wit->value_size, w.nr_ops_total);
+
+	if (w.kv_op_count[CRATE_OP_PUT])	
```

Comment:
Simply to reduce fprintf/if-else ladder as discussed.

Reply:
Done.